packages/core/apps/core-apps-server-internal/src/core_app.ts

```diff
@@ -167,18 +199,80 @@ export class CoreAppsService {
         }
       }
     );
+  }
+
+  private maybeRegisterDynamicConfigurationFeature({
+    config,
+    coreSetup,
+    savedObjectsStart$,
+  }: {
+    config: CoreAppConfig;
+    coreSetup: InternalCoreSetup;
+    savedObjectsStart$: Observable<InternalSavedObjectsServiceStart>;
+  }) {
+    // Always registering the Saved Objects to avoid ON/OFF conflicts in the migrations
+    coreSetup.savedObjects.registerType({
+      name: DYNAMIC_CONFIG_OVERRIDES_SO_TYPE,
```

Comment:
I wonder if we can move this to the Kibana discover feature once it is in Core? Or perhaps we don't want to associate this type of information with that "heartbeat" type polling.

Don't think we should block on this as this seems pretty simple and is a needed feature!

https://github.com/elastic/kibana/pull/187997

Reply:
Yes! Once we have the core implementation clear, we can revisit this :)